openam-core/src/main/resources/Database.section.properties

```diff
@@ -0,0 +1,25 @@
+pluginconfig=sunIdRepoClass
```

Comment:
Please add the license header from the original file with "Portions copyright".

Reply:
Added to all new section properties files.